parachains/runtimes/assets/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -586,7 +605,9 @@ pub mod bridging {
 					.add_equals(AssetHubKusama::get().interior.split_global().expect("invalid configuration for AssetHubKusama").1),
 					// and nothing else
 				BridgeHubPolkadot::get(),
-				None
+				// base delivery fee to local `BridgeHub`
+				// (initially was calculated `51220000` + 10% by test `BridgeHubPolkadot::can_calculate_weight_for_paid_export_message_with_reserve_transfer`)
+				Some((XcmBridgeHubRouterFeeAssetId::get(), 56342000).into())
```

Comment:
Could probably also be moved to BHP primitives and tested there?

Reply:
fixed done + sanity check